tools/doxygen/Doxyfile.c++

```diff
@@ -26,7 +26,7 @@
 # for the list of possible encodings.
 # The default value is: UTF-8.
 
-DOXYFILE_ENCODING      = UTF-8
+DOXYFILE_ENCODING = UTF - 8
```

Comment:
clang-format was too eager to help! Fixed.

On Tue, Jun 23, 2015 at 1:09 PM, Craig Tiller <notifications@github.com>
wrote:

> In tools/doxygen/Doxyfile.c++
> <https://github.com/grpc/grpc/pull/2180#discussion_r33085553>:
>
> > @@ -26,7 +26,7 @@
> >  # for the list of possible encodings.
> >  # The default value is: UTF-8.
> >
> > -DOXYFILE_ENCODING      = UTF-8
> > +DOXYFILE_ENCODING = UTF - 8
>
> Something went horribly wrong here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2180/files#r33085553>.
>